bridge/client/app/_components/_dialogs/ktb-deletion-dialog/ktb-deletion-dialog.component.html

```diff
@@ -9,10 +9,10 @@
     This action cannot be undone. This will permanently delete the <span class="bold">{{ data.name }}</span>
```

Comment:
This will not result in `delete the undefined Subscription`?

Reply:
It showed `... delete the subscription`, but I added a `*ngIf` to the span nonetheless.